### PR TITLE
contributing guidelines

### PR DESCRIPTION
**Pull request by harlo**
readme updated as per https://github.com/freedomofpress/encryption-works/issues/52.

### CODE REVIEW

**Comment by conorsch** [comment]
Nits:
 * "raise an issue" vs. "open an issue"
 * repetitive line numbers for ordered lists can make plain markdown hard to read

Great addition to the README, we can flesh out more guidelines in a CONTRIBUTING.md at a later date.

**Comment by conorsch** [comment]
:+1: lgtm!

File: README.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ This version has been converted to markdown so it can be easily tracked in git a
 
 ## Contributing
 
+If you'd like to see any specific changes or improvements to this document, feel free to [open an issue](https://github.com/freedomofpress/encryption-works/issues) on this repository.
+
+If you want to really get your hands dirty, fork our repository.  Then, if you'd like to contribute to this document, please follow these instructions:
+
+1.	create a *"feature branch"* referencing the issue's ID (i.e. `git checkout -b issue_34`)
+2.	make your presice correction to the document, and commit it.
```
Comment (garrettr) [inline README.md]: Spelling: precise

File: README.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ This version has been converted to markdown so it can be easily tracked in git a
...
+3.	submit a pull request on our repo for consideration.
```
Comment (garrettr) [inline README.md]: Nit on all the lines in this list: sentences should be capitalized.

**Comment by conorsch** [comment]
Any blockers here? Looks like this can get merged whenever.

**Pull request merged**